### ISSUE TITLE
5157e6c commit = no display dropbox icon in tray anymore

### ISSUE THREAD

**Issue by FadeMind** [davidedmundson/xembed-sni-proxy#12]
Lastest git snapshot 5157e6c commit = no display dropbox icon in tray anymore.

**Comment by davidedmundson** [comment]
It definitely wasn't that commit that caused it.
Any chance you can find out which one it was?

**Comment by FadeMind** [comment]
@davidedmundson ok. I will find out. I know that 5157e6c is not a source of issue.

**Comment by FadeMind** [comment]
@davidedmundson 

e8ada875c2b609ba077576fc2e2865d2ce9e9cce failed to build at all: https://pastebin.com/raw.php?i=2p8m8e4W

Commit f989182f0e356ec509e27a7c9e5bd9c42a539b2c build FINE and dropbox icon visible. 

Commits ABOVE f989182f0e356ec509e27a7c9e5bd9c42a539b2c have issue (where e8ada875c2b609ba077576fc2e2865d2ce9e9cce  unable to build anyway). Newest:
7a8ced4dcc948847cfdc5a01f55317da2b4f9547
2caa4442868a57c2e4401d2c6640afeb4681d681
4ccacda41c5ddb96c984398210be46d700f7d311
5157e6c2b7a141a101531ddd06403170576ef808
no display dropbox icon anyway.

tl:dr **WORKING lastest commit is**: f989182f0e356ec509e27a7c9e5bd9c42a539b2c

**Comment by FadeMind** [comment]
FIXED in 392f323f7e15352638ba4090eb6eee3030652220
